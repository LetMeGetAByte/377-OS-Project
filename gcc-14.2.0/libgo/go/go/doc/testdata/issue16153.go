version https://git-lfs.github.com/spec/v1
oid sha256:8c22b0f5062829c3a1d400e8d78b048a924b5f0e6aef830f672717c0f671df80
size 363
