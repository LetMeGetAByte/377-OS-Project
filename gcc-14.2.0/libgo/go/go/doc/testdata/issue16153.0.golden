version https://git-lfs.github.com/spec/v1
oid sha256:40e08bbcd0d4f7e0480a7255d870acf848df1544d12939c6dbfb9c83f0a1fca4
size 277
